pkg/services/resourcesFilterService.go

```diff
@@ -12,6 +14,38 @@ type RootObject struct {
 }
 
 func ShouldResourceBeValidated(admissionReviewReq *admission.AdmissionReview, rootObject RootObject) bool {
+	isResourcesSkipByConfig := configAllowedListsValidation(admissionReviewReq, rootObject)
+
+	if isResourcesSkipByConfig {
+		return false
+	}
+
+	return defaultResourcesValidation(admissionReviewReq, rootObject)
+}
+
+func configAllowedListsValidation(admissionReviewReq *admission.AdmissionReview, rootObject RootObject) bool {
+	namespace := admissionReviewReq.Request.Namespace
+	resourceKind := admissionReviewReq.Request.Kind.Kind
+	resourceName := rootObject.Metadata.Name
+
+	for _, skipListItem := range server.ConfigAllowedLists.SkipList {
+		skipRuleItem := strings.Split(skipListItem, ";")
```

Comment:
so every item in the array has to have 3 `;`?
why not an object instead of a comma separated string?

Reply:
yes, and later it will protected by crd

[https://github.com/datreeio/admission-webhook-datree/pull/148/files#diff-5230fb9571bffe26a62d59d6f3446f4f2204dbaf8bb802462b60e0b83f0b4679](skip)